package.json

```diff
@@ -47,7 +47,8 @@
     "normalize.css": "^8.0.1",
     "rdflib": "^1.0.4",
     "solid-namespace": "^0.2.0",
-    "solid-panes": "^2.0.0"
+    "solid-panes": "^2.0.0",
+    "solid-ui": "^1.1.0"
```

Comment:
Have you tested whether this works with mashlib-dev's `watch` mode, i.e. being able to modify `findProfileImage` in solid-ui and seeing those changes show up in the header bar?

Reply:
I usually run the various watch-commands separately (I like the to spread it out in multiple tabs for (in my opinion) better overview). But I tried it now, and it does work with the "global" watch command in mashlib-dev as well.